infra/templates/azure-paas-single-region/terraform.tfvars

```diff
@@ -0,0 +1,9 @@
+# GENERAL
+resource_group_location = "eastus"
+name = "cobalt-az-simple"
+# SERVICE PLAN
+app_service_name = {
+    cobalt-backend-api = "msftcse/cobalt-paas-single-region:release" 
+}
+# ACR image deploy source
+acr_build_git_source_url = "https://github.com/erikschlegel/echo-server.git"
```

Comment:
Any way to pull this into this repository? Would hate for this project to stop working if you delete this personal repo